### DISCUSSION TITLE
await pb.health.check() showing error in flutter web.

### DISCUSSION THREAD

**Discussion by sabikrahat** [pocketbase/pocketbase#4461] [Q&A]
@ganigeorgiev whenever I call `await pb.health.check()` in `flutter web` it is showing an error `TypeError: Instance of 'DomIterator': type 'DomIterator' is not a subtype of type 'Iterator<any>'}`

**Comment by ganigeorgiev** [comment]
_Please don't ping for no reason. I cannot provide dedicated support._

I'm not sure how to reproduce it.

If you use a custom `httpClientFactory` (eg. `fetch_client`) then it is possible to be something related to that, I'm not sure.

If you need further help, please create a **minimal** reproducible repo or at least provide a more detailed code sample showing how you initialize the SDK and how you are sending the request (_are you sure that the exception is from the pb.health call?_).

**Reply by ganigeorgiev** [reply]
If you use `fetch_client` make sure that you use the latest version as it could be related to https://github.com/Zekfad/fetch_client/issues/9.
